tests/e2e/send-message.js

```diff
@@ -398,26 +386,4 @@ describe('Send message', () => {
     });
   });
 
-  describe('Display message without contact', () => {
```

Comment:
> i officially give up

lol

Reply:
Yeah... it's been a constant source of flakiness despite numerous attempts. This was the final straw for me  :(